code/modules/power/singularity/singularity.dm

```diff
@@ -349,6 +374,14 @@
 	return
 
 
+/obj/singularity/proc/combust_mobs()
+	for(var/mob/living/carbon/C in orange(10, src))
+		C.visible_message("<span class='warning'>[C]'s skin bursts into flame!</span>", \
+						  "<span class='boldannounce'>You feel an inner fire as your skin is suddenly covered in fire!</span>")

```

Comment:
This is a bit of an awkward sentence to read. 

"You feel your insides boil as your skin is engulfed in flame!" would be a better alternative.